com.deflemask.DefleMask.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: "com.deflemask.DefleMask"
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: "/app/share/DefleMask/DefleMask"
+separate-locales: false
+tags:
+  - proprietary
+
+build-options:
+    prepend-pkg-config-path: "/app/lib32/pkgconfig:/usr/lib/i386-linux-gnu/pkgconfig"
+    ldflags: "-L/app/lib32"
+    prepend-path: "/usr/lib/sdk/toolchain-i386/bin"
+    env:
+        CC: "i686-unknown-linux-gnu-gcc"
+        CXX: "i686-unknown-linux-gnu-g++"
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=wayland"
+  - "--socket=pulseaudio"
+  - "--filesystem=home"
+  - "--device=dri"
+
+sdk-extensions:
+  - 'org.freedesktop.Sdk.Compat.i386'
+  - 'org.freedesktop.Sdk.Extension.toolchain-i386'
+
+add-extensions:
+    'org.freedesktop.Platform.Compat.i386':
+        directory: "lib/i386-linux-gnu"
+        version: '19.08'
+
+    'org.freedesktop.Platform.GL32':
+        directory: "lib/i386-linux-gnu/GL"
+        version: '1.4'
+        versions: '19.08;1.4'
+        subdirectories: true
+        no-autodownload: true
```

Comment:
You probably want the opposite, i.e. GL32 extension o be installed automatically?

Reply:
Sorry, that was my mistake, `no-autodownload: true` here was correct - `download-if` will trigger download.